sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java

```diff
@@ -1303,4 +1307,26 @@ private void saveCasesFromLineListing(LineListingLayout lineListingForm, List<Li
 		lineListingForm.closeWindow();
 		ControllerProvider.getCaseController().navigateToIndex();
 	}
+
+	public VerticalLayout getCaseViewTitleLayout(CaseDataDto caseData) {
+		VerticalLayout titleView = new VerticalLayout();
+		titleView.addStyleName(CssStyles.LAYOUT_MINIMAL);
+		titleView.setSpacing(false);
+
+		Label diseaseLabel = new Label(Disease.valueOf(caseData.getDisease().getName()).toString());
+		CssStyles.style(diseaseLabel, CssStyles.H3, CssStyles.VSPACE_NONE, CssStyles.VSPACE_TOP_NONE);
+		titleView.addComponents(diseaseLabel);
+
+		Label classificationLabel = new Label(CaseClassification.valueOf(caseData.getCaseClassification().getName()).toString());
```

Comment:
This looks overcompliated to me; caseData.getCaseClassification().toString() should do the same thing (also check other occurrences of this pattern)